src/main/groovy/org/whispersystems/witness/WitnessPlugin.groovy

```diff
@@ -52,7 +52,7 @@ class WitnessPlugin implements Plugin<Project> {
                 println "dependencyVerification {"
                 println "    verify = ["
 
-                project.configurations.compile.resolvedConfiguration.resolvedArtifacts.each {
+                project.configurations.compile.resolvedConfiguration.resolvedArtifacts.toSorted{it.moduleVersion.id.group}.each {
```

Comment:
Sorting just by groupId is not enough. Please sort first by groupId, then by artifactId.

Reply:
agreed. fixed.